.ruby-version

```diff
@@ -0,0 +1 @@
+jruby-9.1.9.0
```

Comment:
This should be jruby-9.1.10.0, our jenkins environment has that version installed.